themes/doks/layouts/index.html

```diff
@@ -176,19 +172,19 @@ <h3 class="card-title">Corda 4</h3>
         <div class="card no-border h-100">
           <div class="card-body">
             <h3 class="card-title">API reference</h3>
-            <p>Find complete documentation reference in Javadoc and KDoc format for all publicly exposed APIs in the Corda 5 Developer Preview and the Corda 4
-             series (Corda open source 4.x, Corda Enterprise 4.x, and Corda Enterprise Network Manager 1.x).</p>
+            <p>Javadocs and KDocs for all publicly-exposed APIs in the Corda 5 Developer Preview and Corda 4 series
+              (Corda OS, Corda Enterprise, and Corda Enterprise Network Manager (CENM).</p>
```

Comment:
```suggestion
              (Corda OS, Corda Enterprise, and Corda Enterprise Network Manager (CENM)).</p>
```
Nested parentheses don't look great. Not sure if they're technically okay to use? Should it be [CENM] instead?